### DISCUSSION TITLE
Clinical endpoint derivation for oncology study

### DISCUSSION THREAD

**Discussion by ngewkokyew** [MIT-LCP/mimic-code#1379] [MIMIC-General] [answered]
From Gao et al (2021) study, MIMIC database has ~6000+ patients diagnosed with cancer. I feel interested to conduct a retrospective study based on this database. 

After watching the video tutorials, I understand it should be no problem to derive time-to-event endpoint like overall survival. However, I am not sure whether this database captures sufficient parameters to derive other surrogate endpoints such as progression free survival, relapse free survival, and overall response rate etc?

Gao et al. 2021. Characteristics and clinical subtypes of cancer patients in the intensive care unit: a retrospective observational study for two large databases.

**Comment by alistairewj** [comment, marked as answer]
We don't systematically capture concepts like progression free survival. The only long term outcome is mortality.
